.github/workflows/ci.yaml

```diff
@@ -499,19 +499,19 @@ jobs:
         working-directory: site
 
   test-e2e:
-    runs-on: ${{ github.repository_owner == 'coder' && (matrix.variant.enterprise && 'depot-ubuntu-22.04' || 'depot-ubuntu-22.04-4') || 'ubuntu-latest' }}
```

Comment:
just curious why this is not necessary anymore?

Reply:
the premium adds more tests, it just doesn't make sense to run it on a smaller, slower, less stable runner. idk why it was set up like this to begin with.